examples/cmake_hello_c/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@
 cmake_minimum_required(VERSION 3.12)
 cmake_policy(SET CMP0074 NEW)
 project(cmake_hello_c)
+set (CMAKE_CXX_STANDARD 14)
```

Comment:
This will cause problems.  If the default compiler supports something new, e.g. C++20 and we force 14, it will break the standard library and lots of other things.

The way we do it in configure is we only do this if it's required to compile (that is if the compiler is older than 14).  Not sure what the cmake equiv is but presumably there's something.

Reply:
I suppose we could pass those flags from `./configure` to the CMake test. Not sure if you want to make that dependency though.